### ISSUE TITLE
ambient light hides captions

### ISSUE THREAD

**Issue by AlexM-oceano** [extesy/hoverzoom#897]
No matter what i set, i can't get captions visible with ambient light enabled.

on latest Chrome on win10.

**Comment by GrosPoulet** [comment]
Well, i think the original purpose of ambient light was to focus on the picture and only the picture, hiding everything else.
@extesy do you think we should change this ?

**Comment by extesy** [comment]
Yes, the idea of ambilight is to seamlessly extend the picture directly from its borders, so inserting the caption would defeat the purpose. I guess we could try to show captions _in front_ of the ambilight using a shadow border to separate characters from the background. Not sure, I'll need to see how it looks to decide if it's a good idea or not.

**Comment by AlexM-oceano** [comment]
Ah but i love both features. The ambient light looks awesome but i also enjoy seeing the captions. 

People could just not enable both options on if they don't like it, right?
